nautobot/docs/user-guide/platform-functionality/approval-workflow.md

```diff
@@ -1 +1,345 @@
+
+### ApprovalWorkflow
+
+A concrete instance of a workflow triggered for a specific object under review.
+
+- Automatically changes to `Denied` if any stage is denied.
+- Automatically changes to `Approved` if all stages are approved.
+- Calls object’s `on_workflow_approved()` or `on_workflow_denied()` when final state is reached.
+
+### ApprovalWorkflowStage
+
+A concrete instance of stage within a workflow execution. Progresses from `Pending` → `Approved`/`Denied`.
+
+- Approves automatically when minimum approvals are met.
+- Denies immediately if any denial is submitted.
+- Triggers parent workflow state updates.
+
+### ApprovalWorkflowStageResponse
+
+A single user's input on a specific stage - this may be an explicit decision (`approve`/`deny`) or simply a comment without a decision.
+
+- Saving a response can trigger stage and workflow state updates.
+
+## How-To Guides
+
+### Attach Workflow to a Model Instance
+
+Workflows are automatically attached after creating, running, or updating an object that complies with the workflow model and constraints. Manual attachment is not available.
+
+### Using Approval Workflow via UI
+
+#### Create an Approval Workflow Definition with stages
+
+1. Go to `Approvals > Workflow Definitions > Add Approval Workflow Definition`.
+    |   |   |
+    |---|---|
+    | ![Navigation](../../media/development/approval-workflows/nav-approval-workflow-definition.png) | ![Workflow Definition](../../media/development/approval-workflows/approval-workflow-definition.png) |
+
+    ![Add Approval Workflow Definition](../../media/development/approval-workflows/add-approval-workflow-definition.png)
+
+2. Enter:
+    - **Name** (e.g., "Scheduled Job Run Workflow").
+    - **Model** (e.g., `extras|scheduled job`).
+    - **Constraints** (optional JSON filter, e.g., `{"job_model__name": "Bulk Delete Objects"}`, see [Contraints](#constraints)).
+    - **Priority** (only one workflow definition (the highest, i.e. smallest priority one) will apply to any given object under review, see [Priority](#priority)).
+3. In the **Approval Workflow Stage Definitions** section, define one or more stages:
+    - **Stage Name** (a descriptive name for the approval step (e.g., "Finance Approval", "Security Review", "Management Approval")).
+    - **Weight** (order in which the stage is executed).
+    - **Minimum Approvers** (number of approvals required).
+    - **Approver Group** (group of users eligible to approve)
+    - **Denial Message** (optional message shown if denied).
+
+#### Approve or Deny a Stage
+
+1. Go to `Approvals > Approval Dashboard` or on the right `User dropdown menu > Approval Dashboard` and select the **My Approvals** tab.
+    |   |   |
+    |---|---|
+    | ![Navigation1 Dashboard](../../media/development/approval-workflows/nav-approval-dashboard1.png) | ![Navigation2 Dashboard](../../media/development/approval-workflows/nav-approval-dashboard2.png) |
```

Comment:
This isn't rendering correctly either. If you want to be able to put an ordered list or other multi-line content inside a table, you may have to use raw HTML instead of Markdown.